SoftLayer/CLI/nas/detail.py

```diff
@@ -0,0 +1,18 @@
+import click
+import json
+
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+from SoftLayer import utils
+
+NAS_MASK = 'id,username,serviceResourceBackendIpAddress'
+
+
+@click.command()
+@click.argument('nasid', type=int)
+@environment.pass_env
+def cli(env, nasid):
+    account = env.client['Account']
+    for nas in account.getNasNetworkStorage(mask=NAS_MASK):
+        if int(nas['id']) == nasid:
+            env.fout(json.dumps(nas))
```

Comment:
`nasid` sort of implies there's only one nas with that id, so should we break here?